classes/system/salt/master/pkg.yml

```diff
@@ -11,7 +11,3 @@ parameters:
           source: "deb [arch=amd64] http://apt.tcpcloud.eu/nightly/ trusty tcp-salt"
           architectures: amd64
           key_url: "http://apt.tcpcloud.eu/public.gpg"
-      package:
-        salt-formula-galera:
```

Comment:
Will we install this package if we remove completely these lines? Is it better to remove only the line #16?